core/src/test/resources/test_assets.json.quotes_not_supported

```diff
@@ -0,0 +1,208 @@
+{
+  "assets": [
+    {
+      "schema": "stellar",
+      "code": "USDC",
```

Comment:
💡  It seems like you're only using this file in one place, and it's only used to test if the validation fails when quotes are marked as required but not supported. With that in mind, you can probably reduce this file a lot, by leaving just one asset and only the configurations you need for the test. WDYT?

We already have a few similar files in the project and having all configs in all files is a bit hard to maintain, so I'd go leaner with just what you need for this test.

Also, the file name could be more accurate, with something like `test_assets.json.quotes_required_but_not_supported`.